### ISSUE TITLE
Contact

### ISSUE THREAD

**Issue by S-ed** [rossy/youtube-video-download#1]
Greetings.
Seems like "contact" link on description page makes no action.

**Comment by rossy** [comment]
Yeah, I had to release the script before I finished the webpage. There's supposed to be a list of supported languages there also.

**Comment by S-ed** [comment]
I can help with Russian if needed.
